integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1275,15 +1283,18 @@ private void initialize(Map<String, Object> inputDomainMap) throws Exception {
             "FAILURE: command " + cmd + " failed, returned " + result.stderr());
       }
       // create secret for custom sit config t3 public address
+      // create datasource secret for user and password
       cmd =
           "kubectl -n "
               + domainNS
               + " create secret generic "
               + domainUid
               + "-"
-              + "t3publicaddress "
+              + "test-secrets"
```

Comment:
seem confusing : "-" + "test-secrets" - why not just "-test-secrets"?

Reply:
changed to -test-secrets